### DISCUSSION TITLE
Skipped SPs are always reported as `Unknown`

### DISCUSSION THREAD

**Discussion by roadhero** [martomi/chiadog#224] [Q&A] [answered]
**Skipped SPs in reports are always displayed as Unknown**

Running the latest versions of Chiadog, Chia and Python/pip/requirements on a farmer+harvester machine. No issues while running chiadog itself and parsers are indeed launched. Skipped SPs are always displayed as Unknown in reports, and even though there are no issues indeed, I guess the reports should display None? Reporting as a bug, as I'm pretty sure there are no issues with the setup itself:

**Environment:**
 - OS: Windows 10
 - Python version: 3.9.5
 - PIP version: 21.1.2
 - Chia version: 1.1.6

**Comment by martomi** [comment]
This is expected when running on farmer/harvester. The signage point information is only available on the full node.

**Reply by roadhero** [reply]
@martomi Thanks for the follow up, appreciate you looking into the issue. I'm running a full node though - Chia 1.1.6 is launched, synced and farming 24/7. In parallel, I'm plotting on the same machine using swar (M1). There's another plotting machine (M2) in the network, which is only plotting using swar (full node not running), and supplying plots via a network mapped drives to M1.

I've seen this reported by other users as well. Should we try to investigate further? Happy to supply all and any additional details. Thank you!

**Reply by martomi** [reply]
You could verify if the `debug.log` that chiadog has access to, contains logs [in this format](https://github.com/martomi/chiadog/blob/main/tests/chia_log/logs/finished_signage_point/nominal.txt). If it does and the report is still "Unknown" even on the latest version of chiadog (0.6.0), that would indeed indicate some issue.

**Reply by roadhero** [reply]
@martomi Just exported the Select-String from the logs, searching by `Finished signage`, here's how they look:

```2021-06-05T21:04:19.932 full_node chia.full_node.full_node: INFO       Finished signage point 41/64: CC:
065a384f2ba16d3ef65d65f6dd3b146e0d31fba2cece4b1404340994af07d4ce RC:
06620b823f80498af8b417746823fc4f4b928fc11f75880f6d4309daffc1be81
2021-06-05T21:04:30.638 full_node chia.full_node.full_node: INFO       Finished signage point 42/64: CC:
37f507b2d8c55ed5475b97f7042140be726a7979388081b4059f04c8dad31cfd RC:
d182ad5395b6d6a90b6d0dd5994d0a433d786e8c5d29bbdd1723e722a6562b72
2021-06-05T21:04:39.924 full_node chia.full_node.full_node: INFO       Finished signage point 43/64: CC:
cafb1182b40482328f7dc206058ab3572040fcbe692b614e0e7a4c700eb5bf7e RC:
35f579631321287f3b920fac858cce77dd658fa3b20f2a9c00076833cd70ab16
2021-06-05T21:04:48.139 full_node chia.full_node.full_node: INFO       Finished signage point 44/64: CC:
6f0ed1357220496a8616129f4495959c2e595eb8fdb199600e3369ae3de7aa87 RC:
b5b1f3c543acca911c18c4f64663007c483e18807f08617b9f7ecae73555d8b0
2021-06-05T21:04:58.252 full_node chia.full_node.full_node: INFO       Finished signage point 45/64: CC:
1e3d57305984c916a719c72b07e90dda994a170b20ada2c9bf018eceb7215730 RC:
4ec785282b91908cc73506fd51d494e7fff41303ff92b0506a328ae70f94a30a
2021-06-05T21:05:07.579 full_node chia.full_node.full_node: INFO       Finished signage point 46/64: CC:
2bdc648b2db81d62f382c9d8f31060f19a72e53d2e8ab48eaa88c6f508f3c6a6 RC:
674329b4b706eff73064ef538bf62fb3d29fbcdad47cd9c9126edfc1dd2545bd
2021-06-05T21:05:15.648 full_node chia.full_node.full_node: INFO       Finished signage point 47/64: CC:
9dc06387bf1ecb941fc317db2b7a2c30878bf6b3e2cd2c86a695649792cd65d6 RC:
07cf5126fc2423bfcba2da66fb192069d3f793c2b191c0db5dc7bfb0752d48a4
2021-06-05T21:05:23.837 full_node chia.full_node.full_node: INFO       Finished signage point 48/64: CC:
2b018d82f2578557478077c91b0e4c70bcd01a79d01101adcfda1c4ddc24ac55 RC:
3da57cd64e3c911ac849b2f3c2df1530bd7baf8ec2206cd0e60923ac4870cf70
2021-06-05T21:05:32.396 full_node chia.full_node.full_node: INFO       Finished signage point 49/64: CC:
c5c9027dc777b1deca0da5c89b49669e35bda3bdd24d74b8799a9165da1d7f46 RC:
b37a7196386b51214b1d541a6380b88399d979932f2251a4f9f8d68d2c9eb61b
2021-06-05T21:05:40.695 full_node chia.full_node.full_node: INFO       Finished signage point 50/64: CC:
ff087e77313cc52ad0a9b39eab73a7010a17f3643884e6c48bd179fdfe3ad0fe RC:
3452e038ac0071a937bc5029120df75727ef5e52d0d58dba0b51ad6baf33efe5

**Reply by mcortecchia** [reply]
The same is happening to me. Any news about this issue?

**Comment by pieterhelsen** [comment, marked as answer]
This issue is fixed as part of the #254 pull request
